### DISCUSSION TITLE
HTTPS issues in CP

### DISCUSSION THREAD

**Discussion by kylef-fotf** [statamic/cms#9226] [Troubleshooting] [answered]
Hello,

When I try to access my entries in a collection, I get the error "Something went wrong" and nothing loads. Opening the console I can see mixed content errors ("blocked loading mixed active content"). My .env has APP_URL set to an HTTPS domain, and I'm not sure what else I should be doing to tell statamic to load over HTTPS. I found some old documentation about statamic 2 saying to modify the system.yaml file, but I don't see an equivalent in statamic 4.

Below is what I see when trying to load entries over HTTPS. 
![Untitled-1](https://github.com/statamic/cms/assets/104449622/fe7af0ee-0606-4d80-bab2-ecece07887b2)

Going one layer up, it is still able to see how many entries I have in my collections.
![image](https://github.com/statamic/cms/assets/104449622/55b01ac1-cc40-4064-984c-e02397018c32)

Thanks for any insight. I enjoy working with statamic but I am still learning.

-Kyle

**Comment by duncanmcclean** [comment]
Are you using Cloudflare or sitting in front of any reverse proxies?

**Reply by kylef-fotf** [reply]
Yes I am using Cloudflare.

**Reply by duncanmcclean** [reply, marked as answer]
Try installing this package and see if it helps: https://github.com/monicahq/laravel-cloudflare

**Reply by kylef-fotf** [reply]
Thanks, this seems to be working.

Do you know what is happening that makes this package necessary?

**Reply by duncanmcclean** [reply]
When you have the "proxy" enabled on the Cloudflare side, all requests technically go from your user-> cloudflare, then cloudflare -> your server. 

This causes issues because when you configure HTTPS on the Cloudflare side, your server doesn't know about it so it continues to return some URLs as `http://`. I believe the Cloudflare package fixes that by configuring "trusted proxies" so your site knows that HTTP is being used.

_^ I believe that's why it's needed anyway. Some of that might be wrong._
